pkgs/development/libraries/libopendmarc/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, libmilter }:
+
+stdenv.mkDerivation rec {
+  pname = "libopendmarc";
```

Comment:
This is not just a library, but also contains useful standalone programs. So, it is probably better to call this derivation `opendmarc`. I propose to move it to `pkgs/tools/networking/mail/opendmarc`. Then you could also add

```
  outputs = [ "bin" "dev" "out" ];
```

After the `src` attribute, so that e.g. exim doesn't get the headers or binaries in its closure. Or that if someone installs the binary output that they don't get the headers.

Reply:
Splitting leads to a dependency cycle, because something in `share/doc` in `out` references stuff in `bin`.

Reply:
It's only in the documentation. So, the references could be removed, or you could make documentation a separate output:

```
  outputs = [ "bin" "dev" "out" "doc" ];
```

In this case that would be the most elegant solution.

Reply:
done. I think this will mainly be used as a library, which is why I chose `development/libraries` instead of `tools`. Lots of "libraries", e.g. openssl, ship tools and example binaries, but they're still in this folder.

Reply:
Ack on the directory. But upstream calls the project `opendmarc` and not `libopendmarc`, so to me `opendmarc` is more appropriate as `pname` and the directory name (thus `development/libraries/opendmarc`).

Reply:
:+1: I also changed the name of the attribute in `all-packages.nix`.